packages/.eslintrc.json

```diff
@@ -25,6 +25,15 @@
         "import/no-deprecated": "warn",
         "lodash/import-scope": ["error", "member"],
         "unused-imports/no-unused-imports-ts": "warn",
+        "unicorn/better-regex": "error",
```

Comment:
I'm okay with other rules, but let's wait for the uppercase problem in the RegExp

Reply:
i not think the is problem

Reply:
![image](https://user-images.githubusercontent.com/5390719/128815637-d46ad372-52e7-40f0-a641-71c07075e305.png)

I found this rule doesn't affect the case. I'll revert those changes in the letter case.